workspace/apps/pidp/src/app/features/auth/pages/login/login.page.html

```diff
@@ -12,44 +12,47 @@
 
       <section class="welcome-section">
         <div class="welcome-section-container">
-          <div *ngIf="banners.length > 0" class="banner">
+          <div *ngIf="banners.length > 0; else welcomesection">
             <app-banner [banners]="banners"></app-banner>
           </div>
-          <div *ngIf="banners.length === 0" class="welcome-container">
-            <div class="title-content-container">
-              <h1 class="title">
-                <span class="title-text">
-                  <strong>OneHealthID Service</strong>
-                </span>
-              </h1>
-              <section class="title-site-description">
-                <p>
-                  A suite of identity-related products and services for
-                  healthcare providers in BC.
-                </p>
-              </section>
-            </div>
-            <ng-container *ngTemplateOutlet="login"></ng-container>
-          </div>
-          <div *ngIf="banners.length === 0" class="links">
-            <app-need-help
-              customClass="need-help-large"
-              [showIcon]="true"></app-need-help>
-            <div class="about">
-              <a
-                uiAnchor
-                href="https://www2.gov.bc.ca/gov/content/health/practitioner-professional-resources/onehealthid-service">
-                About, FAQs, Guides
-                <i class="material-icons-outlined">arrow_circle_right</i>
-              </a>
-            </div>
-          </div>
         </div>
       </section>
     </div>
   </div>
 </ui-layout-header-footer>
 
+<ng-template #welcomesection>
+  <div *ngIf="banners.length === 0" class="welcome-container">
```

Comment:
This second `ngIf` is not necessary as the calculation happens on this line
`<div *ngIf="banners.length > 0; else welcomesection">`

It's like an `If...else` clause.

Reply:
Fixed.